tvb_documentation/manuals/UserGuide/DataExchange.rst

```diff
@@ -275,49 +281,50 @@ ZIP file should include files with the following naming schema and format:
    - contains a matrix of weights
    - any value greater than zero is considered as a connection. You should not have negative values in your weights file.
 
-#. If any file name contains "centres" it will be considered as the container
+2. If any file name contains "centres" it will be considered as the container
```

Comment:
Why hard-code the numbers? 
When left with # placeholder, these will be automatically numbered when the HTML is generated.

Reply:
When I was updating the GUI tutorials, the numberings were mixed (in some places the numbers were hardcoded, in other places there was the # placeholder), so that is why I thought to keep it consistent everywhere.

Reply:
I understand, but then the right consistency will be not to hard-code numbers, and use placeholders in every place possible.